examples/poa-sidechain/package.json

```diff
@@ -0,0 +1,158 @@
+		"start": "echo Run \"./bin/run start\" to start the app",
+		"build": "tsc",
+		"test": "jest --passWithNoTests",
+		"test:coverage": "jest --passWithNoTests --coverage=true --coverage-reporters=text",
+		"test:ci": "jest --passWithNoTests --coverage=true --coverage-reporters=json",
+		"version": "oclif readme --multi --dir=docs/commands && git add README.md docs",
+		"prepublishOnly": "npm ci && npm run lint && npm run build",
+		"app": "./bin/run start --api-http --api-ipc",
+		"init-genesis": "./bin/run genesis-block:create --output config/default --assets-file ./config/default/genesis_assets.json",
+		"updateAuthority": "./node_modules/.bin/ts-node scripts/updateAuthority.ts"
+	},
+	"bin": {
+		"pos-mainchain": "./bin/run"
+	},
+	"lisk": {
+		"addressPrefix": "lsk"
+	},
+	"oclif": {
+		"bin": "poa-mainchain",
+		"commands": "./dist/commands",
+		"plugins": [
+			"@oclif/plugin-help",
+			"@oclif/plugin-autocomplete",
+			"@oclif/plugin-version"
+		],
+		"additionalHelpFlags": [
+			"-h"
+		],
+		"additionalVersionFlags": [
+			"-v"
+		],
+		"topicSeparator": " ",
+		"topics": {
+			"account": {
+				"description": "Commands relating to poa-mainchain accounts."
+			},
+			"block": {
+				"description": "Commands relating to poa-mainchain blocks."
+			},
+			"blockchain": {
+				"description": "Commands relating to poa-mainchain blockchain data."
+			},
+			"console": {
+				"description": "poa-mainchain interactive REPL session to run commands."
+			},
+			"config": {
+				"description": "Commands relating to poa-mainchain node configuration."
+			},
+			"endpoint": {
+				"description": "Commands relating to endpoints."
+			},
+			"keys": {
+				"description": "Commands relating to account, generator and bls keys."
+			},
+			"generator": {
+				"description": "Commands relating to poa-mainchain forging and generator-info data."
+			},
+			"hash-onion": {
+				"description": "Create hash onions to be used by the forger."
+			},
+			"system": {
+				"description": "Commands relating to poa-mainchain node."
+			},
+			"passphrase": {
+				"description": "Commands relating to poa-mainchain passphrases."
+			},
+			"sdk": {
+				"description": "Commands relating to Lisk SDK development."
+			},
+			"transaction": {
+				"description": "Commands relating to poa-mainchain transactions."
+			}
+		}
+	},
+	"files": [
+		"/bin",
+		"/npm-shrinkwrap.json",
+		"/oclif.manifest.json",
+		"/dist",
+		"/config",
+		"/docs"
+	],
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"dependencies": {
+		"@liskhq/lisk-framework-dashboard-plugin": "^0.2.0-alpha.7",
+		"@liskhq/lisk-framework-faucet-plugin": "^0.2.0-alpha.7",
+		"@liskhq/lisk-framework-forger-plugin": "^0.3.0-alpha.7",
+		"@liskhq/lisk-framework-monitor-plugin": "^0.3.0-alpha.7",
+		"@liskhq/lisk-framework-report-misbehavior-plugin": "^0.3.0-alpha.7",
+		"@liskhq/lisk-framework-chain-connector-plugin": "0.1.0-beta.0",
```

Comment:
if you use the same version as pos-mainchain then it will not update the yarn.lock file unnecessarily